src/Core/Content/Property/PropertyGroupDefinition.php

```diff
@@ -71,6 +74,7 @@ protected function defineFields(): FieldCollection
             (new StringField('display_type', 'displayType'))->setFlags(new Required()),
             (new StringField('sorting_type', 'sortingType'))->setFlags(new Required()),
             new BoolField('filterable', 'filterable'),
+            new BoolField('visible_on_detail', 'visibleOnDetail'),
```

Comment:
Please mark this as required

Reply:
but why?

Reply:
ok, it is `NOT NULL` in the migration

Reply:
`filterable` is also NOT NULL (as it was copied from that file). Should that also be made required then?